src/test/java/team4/footwithme/stadium/service/StadiumServiceImplTest.java

```diff
@@ -0,0 +1,192 @@
+package team4.footwithme.stadium.service;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.locationtech.jts.geom.Point;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.transaction.annotation.Transactional;
+import team4.footwithme.global.util.PositionUtil;
+import team4.footwithme.member.domain.*;
+import team4.footwithme.member.repository.MemberRepository;
+import team4.footwithme.stadium.service.response.StadiumDetailResponse;
+import team4.footwithme.stadium.service.response.StadiumsResponse;
+import team4.footwithme.stadium.domain.Stadium;
+import team4.footwithme.stadium.repository.StadiumRepository;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+@SpringBootTest
+@Transactional
+class StadiumServiceImplTest {
```

Comment:
다른분들한테도 드린 리뷰인데 관태님은 `@Transactional` 잘 사용하고 계셨군요

그런데 `@SpringBootTest` 어노테이션을 하나씩 붙여서 사용하는 것도 비용이라고 합니다. 
제가 IntegerationTestSupport라는 추상클래스를 만들어두었는데 이거 활용하셔서 @SpringBootTest 어노테이션 지우고 사용하셔도 좋아요
```suggestion
@Transactional
class StadiumServiceImplTest extends IntegrationTestSupport {
```
이러한 방식으로 바꾸는게 테스트의 비용을 줄일 수 있답니다

[[Spring] 테스트 환경](https://tommykim.tistory.com/87) 이 글 참고해보시면 좋을 것 같아요

Reply:
글 읽어보도록 하겠습니다! 감사합니다!